lib/runners/firecracker.nix

```diff
@@ -11,12 +11,15 @@ let
     interfaces volumes shares devices
     kernel initrdPath
     storeDisk;
+  inherit (microvmConfig.firecracker) cpuConfig;
```

Comment:
I think `config` is rather redundant in an option name. It's called `cpu` for qemu, too (albeit it has a different type).

Reply:
Also, the variable is only used in one line. Move the `let ... in` more locally.